cpp/src/dbscan/adjgraph/runner.cuh

```diff
@@ -39,8 +38,9 @@ void run(const raft::handle_t& handle,
 {
   Pack<Index_> data = {vd, adj, adj_graph, adjnnz, ex_scan, N};
   switch (algo) {
-    // TODO: deprecate naive runner. cf #3414
-    case 0: Naive::launcher<Index_>(handle, data, batch_size, stream); break;
+    case 0:
+      ASSERT(
+        false, "Incorrect algo '%d' passed! Naive version of adjgraph has been removed.", algo);
```

Comment:
This is mostly internal code. Since this has been reduced to only a single option, I propose we remove the switch statement altogether (as well as the `algo` argument to the launcher).

Reply:
Agreed. I discussed this with @tfeher. Given the upcoming release we thought it would be better to do this in a seperate PR and merge into the next release branch 👍